scripts/test-extensions.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+#
+# Checkout extensions to a temporary directory and run tilt against them.
+
+set -ex
+
+DIR=$(mktemp -d)
+git clone git@github.com:tilt-dev/tilt-extensions "$DIR"
```

Comment:
FYI You might want to do `https://github.com/tilt-dev/tilt-extensions` so it'd work without an SSH key, but not a big deal

Reply:
good point!